nodecore-spv/src/main/java/veriblock/service/impl/Blockchain.java

```diff
@@ -125,7 +129,12 @@ public void addAll(List<VeriBlockBlock> blocks) throws SQLException {
             BigInteger work = blockWorks.get(veriBlockBlock.getPreviousBlock().toString());
             //This block is from fork, our Blockchain doesn't have this previousBlock.
             if(work == null){
-                continue;
+                StoredVeriBlockBlock storedVeriBlockBlock = blockStore.get(veriBlockBlock.getPreviousBlock());
+                if(storedVeriBlockBlock == null) {
```

Comment:
```suggestion
                if (storedVeriBlockBlock == null) {
```
Doesn't the editor config work on your IDE?

Reply:
I will check it.